content/project/governance-meeting/index.adoc

```diff
@@ -37,13 +37,12 @@ Everyone is welcome to add a topic for one of the incoming meetings by suggestin
 
 === Archived Meetings
 
-Before using HackMD, we used to use Google Docs for the agenda.
+Before using HackMD, we used Google Docs for the agenda.
 Previous notes are available in the old link:https://docs.google.com/document/d/11Nr8QpqYgBiZjORplL_3Zkwys2qK1vEvK-NYyYa4rzg[Google doc].
 
-Older agendas, minutes and IRC logs are archived on these pages:
-
-Minutes and raw conversation history for each meeting can be found in full on link:http://meetings.jenkins-ci.org/[meetings.jenkins-ci.org].
+Agendas, minutes, and other notes are archived on these pages:
```

Comment:
I think meetings since Oct 30 are not currently discoverable from this page?

Reply:
Maybe I'm not understanding the page you mean when you say "this page".

I see all the [2023 meeting notes](https://github.com/jenkins-infra/governance-meetings-archives/blob/main/docs/2023.md) and all the [2024 meeting notes](https://github.com/jenkins-infra/governance-meetings-archives/blob/main/docs/2024.md) in the [GitHub repository](https://github.com/jenkins-infra/governance-meetings-archives/).

I've lost write permission to the [Google Doc](https://docs.google.com/document/d/11Nr8QpqYgBiZjORplL_3Zkwys2qK1vEvK-NYyYa4rzg), so I had to submit changes as a proposal with a request to @oleg-nenashev that he grant me write access to the Google Doc.  I had write access previously, but that access has apparently been changed to only allow me to suggest content.

Reply:
@kmartens27 noted a place in the governance documentation that is unclear.  I've proposed an improvement in:

* #7246